src/utils/common.ts

```diff
@@ -89,3 +92,16 @@ export const buildUrlOptions = (options: { [key: string]: string }): string => {
   });
   return optionStr;
 };
+
+export const getOfferItemIndexByBestOffer = (gasUsd: (number | undefined)[], recieveAmount: number[]) => {
+  let index = 0;
+  let minAmount = gasUsd[0] ? recieveAmount[0] - gasUsd[0] : 100000;
+
+  for (let i = 1; i < gasUsd.length; i++) {
+    let gasAmount = gasUsd[i];
+    if (gasAmount) {
+      index = recieveAmount[i] - gasAmount > minAmount ? i : index;
+    }
+  }
```

Comment:
I think this looks easier to read instead of having an instance where we write `index = index` 
```suggestion
    if (gasAmount && (recieveAmount[i] - gasAmount) > minAmount) index = i;
  }
```